sdk/api_fingerprint.go

```diff
@@ -21,6 +21,134 @@ import (
 
 type FingerprintApiService service
 
+/*
+FingerprintApiService Get event by request ID
+This endpoint allows you to get events with all the information from each activated product - BOTD and Fingerprinting. Use the requestId as a URL path :request_id parameter. This API method is scoped to a request, i.e. all returned information is by requestId.
+  - @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+  - @param requestId Request ID
+
+@return EventResponse
+*/
+func (a *FingerprintApiService) GetEvent(ctx context.Context, requestId string) (EventResponse, *http.Response, error) {
+	var (
+		localVarHttpMethod  = http.MethodGet
+		localVarPostBody    interface{}
+		localVarFileName    string
+		localVarFileBytes   []byte
+		localVarReturnValue EventResponse
+	)
+
+	// create path and map variables
+	localVarPath := a.client.cfg.basePath + "/events/{request_id}"
+	localVarPath = strings.Replace(localVarPath, "{"+"request_id"+"}", fmt.Sprintf("%v", requestId), -1)
+
+	localVarHeaderParams := make(map[string]string)
+	localVarQueryParams := url.Values{}
+	localVarFormParams := url.Values{}
+
+	localVarQueryParams.Add("ii", "fingerprint-pro-server-go-sdk/1.0.2")
```

Comment:
Is it possible to get current library version dynamically instead of using `"fingerprint-pro-server-go-sdk/1.0.2"` as hardcoded value?

Reply:
It's not hardcoded in fact. This file is generated from a template stored in `template/api.mustache`. We read the version from `config.json` when we generate the code.